json/src/test/java/com/basistech/rosette/apimodel/ModelTest.java

```diff
@@ -251,12 +250,12 @@ private Object createObjectForType(Class<?> type, Type genericParameterType, boo
 
         case "Collection":
         case "List": {
+            List<Object> list = new ArrayList<>();
```

Comment:
Empty fields (list, map, set, etc..) are null, like ADM. Let's not change the assumption.